src/Illuminate/Notifications/Messages/SlackAttachment.php

```diff
@@ -260,4 +281,21 @@ public function timestamp($timestamp)
 
         return $this;
     }
+
+    /**
+     * Set the author.
+     *
+     * @param  string $name
+     * @param  string $link
```

Comment:
string|null

Reply:
@GrahamCampbell Updated, I also updated another place missing `null` in line 120, let me know if you want me separate that change to a new PR.